### DISCUSSION TITLE
Localization Support

### DISCUSSION THREAD

**Discussion by AmadeusGraves** [TriliumNext/Trilium#3809] [General]
One issue that I see with the note-taking app Trillium is that it does not have localization support. Localization is the process of adapting a product or service to meet the language of a particular country or region. Without localization support, Trillium may not be accessible  for non-English speaking users.

Localization support can also help Trillium reach a wider audience and increase its user base. By providing translations and adapting the app to different regions, Trillium can attract users from different countries who may not have otherwise used the app.

**Comment by AmadeusGraves** [comment]
I have seen through the documentation and the project that there is nothing about support for the translation of the application. It is possible that internationalization is not present. Could someone tell me if it will be implemented? I would like to contribute to the application by translating it into Spanish.

**Comment by zadam** [comment]
Hello, [localization is currently not planned](https://github.com/zadam/trilium/wiki/FAQ#translation--localization-support). Maybe in the farther future.
